ajax/libs/vanilla-lazyload/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "vanilla-lazyload",
+  "filename": "dist/lazyload.min.js",
+  "version": "2.1.2",
+  "description": "LazyLoad is a fast, lightweight and flexible script for loading images only when they're about to enter the viewport of a scrollable area, with an excellent support to the progressive JPEG image format. Typescript module definition is available.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/verlok/lazyload.git"
+  },
+  "license": "MIT",
+  "authors": [
+    "Andrea",
+    "verlok",
+    "Verlicchi"
+  ],
+  "keywords": [
+    "lazy load",
+    "progressive JPEG",
+    "performance",
+    "no-jquery"
+  ],
+  "homepage": "https://verlok.github.io/lazyload",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/verlok/lazyload.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "+(src|dist)/lazyload*"
+        ]
+      }
+    ]
```

Comment:
@Piicksarn Separate `lazyload*` files doesn't make sense. I suggest you can use this config:
```json
"fileMap": [
  {
    "basePath": "dist",
    "files": [
      "**/*"
    ]
  },
  {
    "basePath": "src",
    "files": [
      "lazyload*"
    ]
  }
]
```
Thanks.